keras/utils/io_utils.py

```diff
@@ -3,6 +3,8 @@
 from __future__ import division
 from __future__ import print_function
 
+import contextlib
+
 import numpy as np
 from collections import defaultdict
```

Comment:
should `collections` and `sys` be in the same import block as `contextlib` since they are builtins?

Reply:
Agree! Doesn't seem to be a clear standard on the ordering of imports though, or is there?